src/ReactDOMServer.re

```diff
@@ -4,3 +4,68 @@ external renderToString: React.element => string = "renderToString";
 [@bs.module "react-dom/server"]
 external renderToStaticMarkup: React.element => string =
   "renderToStaticMarkup";
+
+type error = {.};
+
+[@deriving abstract]
+type options = {
+  [@bs.optional]
+  bootstrapScriptContent: option(string),
+  [@bs.optional]
+  bootstrapScripts: option(array(string)),
+  [@bs.optional]
+  bootstrapModules: option(array(string)),
+  [@bs.optional]
+  identifierPrefix: option(string),
+  [@bs.optional]
+  namespaceURI: option(string),
+  [@bs.optional]
+  nonce: option(string),
+  [@bs.optional]
+  onAllReady: option(unit => unit),
+  [@bs.optional]
+  onError: option(error => unit),
+  [@bs.optional]
+  onShellReady: option(unit => unit),
+  [@bs.optional]
+  onShellError: option(error => unit),
+  [@bs.optional]
+  progressiveChunkSize: option(int),
+};
+
+[@bs.module "react-dom/server"]
+external renderToPipeableStream: (React.element, options) => string =
+  "renderToPipeableStream";
+
+let renderToPipeableStream =
```

Comment:
shouldn't most of these be optionally labeled arguments?

Reply:
Totally, fixed in [`8347855` (#756)](https://github.com/reasonml/reason-react/pull/756/commits/834785584f01af3a95fb19fcf4f2dff6717c5127)